android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -1,5 +1,12 @@
 package com.mattermost.rnbeta;
 
+<<<<<<< HEAD
```

Comment:
Bad merge?

Reply:
oh my bad, the merge is incomplete as i'm moving `react-native-android-open-settings` to lazy load too, please review again, thanks